Graphics/Blank/JavaScript.hs

```diff
@@ -614,8 +610,6 @@ jsEscapeBuilder = fromBuilder . fromLazyText . TL.concatMap jsEscapeChar . toLaz
 jsEscapeChar :: Char -> TL.Text
 jsEscapeChar '\\' = "\\\\"
 -- Special control sequences.
-jsEscapeChar '\0' = jsUnicodeChar '\0' -- Ambigous with numbers
-jsEscapeChar '\a' = jsUnicodeChar '\a' -- Non JS
```

Comment:
Just to make sure I understand this new code correctly, the new catch-all case for `jsEscapeChar` will correctly handle `\0` and `\a`, right?

Reply:
My understanding is that, e.g. for `\0` you could either use `\u0000` (as was before) or `\x00` (as is now and how it's handled in GHCJS). In fact it seems that in GHCJS they just use `\xXX` for anything that fits one byte, but is a control symbol or is not an ASCII symbol.

Reply:
That makes sense, thanks for the explanation.